### ISSUE TITLE
Adrift-Field-Methods Development

### ISSUE THREAD

**Issue by shannonrankin** [SAEL-SWFSC/adrift-field-methods#1]
The 'Adrift-Field-Methods' repository has been initiated using a Quarto Book Template with Webpage (HTML) output. We will shift all documentation related to field efforts FROM google drive TO this repository. This includes hardware (description, assembly, modifications), methods to prepare for deployment, deployment methods, tracking methods, and retrieval methods.  Each 'topic' will have its own *.qmd file stored in the 'content' folder. These need to be modified to include the content currently in the google drive. We may decide we want to modify the sections (split/lump), and we should consider how to address alternative (including past) methods, as well as how we will want to modify these going forward. 

- [x] Improve Headings/contents format to make more readable? (perhaps branched from Hardware/FieldOps?)
- [x] Revise index.qmd
- [x] Revise Hardware-Summary.qmd
- [x] Revise Hardware-PoleBuoy.qmd
- [x] Discuss: How to describe previous/alternative Buoy Configs?
- [x] Revise Hardware-GPS-Solar.qmd
- [ ] Discuss: How to describe Spot Devices?
- [x] Discuss: should hardware tote contents go into the summary or stand-alone?
- [x] Revise Hardware-Tote-Contents.qmd
- [x] Revise Hardware-Array.qmd
- [x] Revise Hardware-ST4300.qmd
- [x] Revise Hardware-ST640.qmd
- [x] Revise Hardware-Sensus-Depth.qmd
- [x] Revise Field-Deployment-Preparation.qmd
- [x] Revise Field-Deployment.qmd
- [x] Revise Field-Metadata.qmd
- [x] Revise Field-Tracking.qmd
- [x] Revise Field-Retrieval.qmd
- [x] Revise Data-Download.qmd
- [ ] Revise Readme (Help user navigate document and let them know how to ask for help)

**Comment by Kourtney-Burger** [comment]
**Responding to this task:**
Discuss: should hardware tote contents go into the summary or stand-alone?

I think it makes sense to have it as its own stand-alone section because not everything lives in the tote (like the pole and GPS) so I think it makes sense that this is a section after the summary. I can add a sentence to the summary that states what is stored in the tote and what is stored separately? Let me know what you think

**Comment by Kourtney-Burger** [comment]
**Responding to this task:**
Revise Field-Metadata.qmd

What are we putting here? The information for deployment details and who to report it to? Or something else?

**Comment by Kourtney-Burger** [comment]
**Responding to this task:**
Revise Data-Download.qmd

Are we putting information here about how to offload data from the recorders and depth sensors or something else?

**Comment by Kourtney-Burger** [comment]
**Responding to this task:**
Revise Field-Tracking.qmd

Is this for tracking with the GPS, like driftwatch information?

**Comment by Kourtney-Burger** [comment]
**Responding to this task:**
Discuss: How to describe previous/alternative Buoy Configs?

Would it make sense to have another page within the hardware section that breaks down the previous buoy configs and lists the updates we made and why we made them?

**Comment by shannonrankin** [comment]
Note to self re: learning how to effectively use Issues.... It is best to separate each task into smaller issues to address comments individually!  Will try to address these individually if I can...

**Comment by shannonrankin** [comment]
> Responding to this task:
Discuss: should hardware tote contents go into the summary or stand-alone?
I think it makes sense to have it as its own stand-alone section because not everything lives in the tote (like the pole and GPS) so I think it makes sense that this is a section after the summary. I can add a sentence to the summary that states what is stored in the tote and what is stored separately? Let me know what you think

**Yes, ok.** I think we will ultimately re-organize things a bit. Now that we know how to add sections, we use those to help navigate information. But one step at a time--- yes agreed for Stand-Alone.

**Comment by shannonrankin** [comment]
> **Responding to this task:** Revise Field-Metadata.qmd
> 
> What are we putting here? The information for deployment details and who to report it to? Or something else?

Deployment Details (initial information, and then followup), Inform Kourtney (you) re: possible permit needs (if they will deploy in a sanctuary). If we think of other things, we can add them as we think of them.

**Comment by shannonrankin** [comment]
> **Responding to this task:** Revise Data-Download.qmd
> 
> Are we putting information here about how to offload data from the recorders and depth sensors or something else?

This will include HOW to download data (with links to pages on how to download data from recorder, sensus, etc), and how to save/ship the data to one of us to upload. If you feel information is missing, create an issue & assign to me. It might be stuff I have on my personal drive

**Comment by shannonrankin** [comment]
> **Responding to this task:** Revise Field-Tracking.qmd
> 
> Is this for tracking with the GPS, like driftwatch information?

Related to using the recorders and driftwatch. Create an issue & assign to me.

**Comment by shannonrankin** [comment]
> **Responding to this task:** Discuss: How to describe previous/alternative Buoy Configs?
> 
> Would it make sense to have another page within the hardware section that breaks down the previous buoy configs and lists the updates we made and why we made them?

For now, create a link on the pole buoy page that says it will link to a page with pole buoy alternatives (then make a sample qmd and create an issue for me to do). Do the same for each relevant component (Array?).

**Comment by Kourtney-Burger** [comment]
Trying to limit the messages here but anything that I agreed with/resolved, I added a thumbs up to. 

> > **Responding to this task:** Revise Data-Download.qmd
> > Are we putting information here about how to offload data from the recorders and depth sensors or something else?
> 
> This will include HOW to download data (with links to pages on how to download data from recorder, sensus, etc), and how to save/ship the data to one of us to upload. If you feel information is missing, create an issue & assign to me. It might be stuff I have on my personal drive

In regards to this, I submitted a new issue for you to add the information or links to information on how to do this and I can edit it.

**Comment by shannonrankin** [comment]
> > > **Responding to this task:** Revise Data-Download.qmd
> > > Are we putting information here about how to offload data from the recorders and depth sensors or something else?
> > 
> > 
> > This will include HOW to download data (with links to pages on how to download data from recorder, sensus, etc), and how to save/ship the data to one of us to upload. If you feel information is missing, create an issue & assign to me. It might be stuff I have on my personal drive
> 
> In regards to this, I submitted a new issue for you to add the information or links to information on how to do this and I can edit it.

Yes, I think it would be good to add all info related to downloading the data, so when they get gear on board, they can go directly to that section & be sure to do it all.... Will follow your next issue- thanks!
